cmd/cli/envcli.go

```diff
@@ -5,66 +5,66 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/smartcontractkit/helmenv/environment"
+	"github.com/smartcontractkit/helmenv/tools"
 	"github.com/urfave/cli/v2"
 	"os"
 )
 
 func init() {
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr}).Level(zerolog.InfoLevel)
+}
+
+var presetFlag = &cli.StringFlag{
+	Name:     "preset",
+	Aliases:  []string{"p"},
+	Usage:    "filepath to the environment preset",
+	Required: true,
+}
+
+var environmentFlag = &cli.StringFlag{
+	Name:     "environment",
+	Aliases:  []string{"e"},
+	Usage:    "filepath to the environment file",
+	Required: true,
 }
 
 func main() {
 	app := &cli.App{
-		Name:                 "envcli",
-		Usage:                "setup default environment",
-		EnableBashCompletion: true,
-		Flags: []cli.Flag{
-			&cli.StringFlag{
-				Name:     "kubectlProcess",
-				Aliases:  []string{"kctl"},
-				Value:    "/usr/local/bin/kubectl",
-				Usage:    "kubectl process name to use forking for port forwarding",
-				Required: false,
-			},
-			&cli.StringFlag{
-				Name:     "preset",
-				Aliases:  []string{"p"},
-				Usage:    "environment preset name",
-				Required: true,
-			},
-		},
-		Before: func(c *cli.Context) error {
```

Comment:
Is there any other check that completely prevents a user to interact with an env when both `persistent` and `persistent_connection` are set to false?